### DISCUSSION TITLE
Redirect via an action

### DISCUSSION THREAD

**Discussion by dfioravanti** [jowilf/starlette-admin#205] [Q&A]
I have an action that I am trying to implement via the `@action` decorator. This action calls some REST endpoints and gets back a response that is a HTML page that I would like to render in the browser. With flask admin all I need to do is just return the response as a redirect. In starlette-admin it looks like action can return only strings. Am I wrong? or is there a way to redirect via an action?

Thanks =)

**Comment by jowilf** [comment]
You're correct, currently there is no support for custom responses in batch actions. I've opened an issue to address this limitation ( #207 )
